.github/workflows/release.yaml

```diff
@@ -54,6 +54,14 @@ jobs:
           pinata-api-key: ${{ secrets.PINATA_API_KEY }}
           pinata-secret-api-key: ${{ secrets.PINATA_API_SECRET_KEY }}
 
+      - name: Pin to Crust
+        uses: crustio/ipfs-crust-action@v1.0.8
+        continue-on-error: true
+        timeout-minutes: 2
+        with:
+          cid: ${{ steps.upload.outputs.hash }}
```

Comment:
Does this mean the pin is fetched from other ipfs nodes?

Reply:
Yes. The pin will be fetched from other (50+) IPFS nodes. The whole process after this PR will shows like below:
![image](https://user-images.githubusercontent.com/10335300/112086702-a94bc500-8bc7-11eb-9c31-343f6af39bee.png)

PS: the IPFS nodes status of Crust Network can be checked [here](https://telemetry.polkadot.io/#map/Crust%20Maxwell). **You may need Crust seeds to call the action. You can send a mail to hi@crust.network and I will send it over.**

Reply:
Hey @moodysalem , any questions, please feel free to ask me